### PR TITLE
[infra/onert] Rename debian package

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit renames runtime debian package nnfw -> onert.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: infra/debian/runtime/control [hunk 1]
```diff
@@ -6,20 +6,40 @@ Build-Depends: cmake, debhelper (>=9)
...
 Description: one-runtime package
 
-Package: nnfw-dev
+Package: nnfw
+Architecture: all
+Priority: optional
+Section: oldlibs
+Depends: onert, ${shlibs:Depends}, ${misc:Depends}
+Description: one-runtime transitional package
+  This is a transitional package. It can safely be removed.
```
[the remark this answers is not in the document]
Comment (jyoungyun) [reply infra/debian/runtime/control:23]: Here is another link: https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-and-replacing-packages-replaces :)

File: infra/debian/runtime/control [hunk 2]
```diff
@@ -6,20 +6,40 @@ Build-Depends: cmake, debhelper (>=9)
...
+Package: onert-dev
+Replaces: nnfw-dev (<< 1.29.0)
+Breaks: nnfw-dev (<< 1.29.0)
 Architecture: amd64
 Multi-Arch: same
-Depends: nnfw, ${shlibs:Depends}, ${misc:Depends}
+Depends: onert, ${shlibs:Depends}, ${misc:Depends}
 Description: one-runtime development package
 
-Package: nnfw-plugin-dev
```
Comment (jyoungyun) [inline infra/debian/runtime/control:21]: (Q) Is it your intention not to create `nnfw-plugin-dev`'s transitional package?

File: packaging/nnfw-plugin.pc.in [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Name: nnfw-plugin
```
Comment (jyoungyun) [inline packaging/nnfw-plugin.pc.in:1]: Maybe you should change the name of this file.

File: packaging/nnfw.pc.in [hunk 4]
```diff
@@ -1,5 +1,5 @@
 Name: nnfw
```
Comment (jyoungyun) [inline packaging/nnfw.pc.in:1]: ditto.

File: infra/debian/runtime/control (hunk 2 above)
Comment (hseok-oh) [reply infra/debian/runtime/control:21]: Right. Nobody is using `nnfw-plugin-dev` now.

File: packaging/nnfw-plugin.pc.in (hunk 3 above)
Comment (hseok-oh) [reply packaging/nnfw-plugin.pc.in:1]: I've updated `rules` to update name after installation.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**